keyboards/planck/keymaps/buffet/keymap.c

```diff
@@ -0,0 +1,384 @@
+/* Copyright 2015-2017 Jack Humbert
+ *           2020 Niclas Meyer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#define K(kc) (1ull<<((kc) - ALPHA))
+
+#define KALPHA   K(ALPHA)
+#define KBETA    K(BETA)
+#define KGAMMA   K(GAMMA)
+#define KDELTA   K(DELTA)
+#define KEPSILON K(EPSILON)
+#define KZETA    K(ZETA)
+#define KTHETA   K(THETA)
+#define KIOTA    K(IOTA)
+#define KKAPPA   K(KAPPA)
+
+enum keys {
+    ALPHA = SAFE_RANGE,
+    BETA,
+    GAMMA,
+    DELTA,
+    EPSILON,
+    ZETA,
+    THETA,
+    IOTA,
+    KAPPA,
+    LOCK,
+    NONE,
+};
+
+enum layers {
+    NORMAL,
+    QWERTY,
+    GAME,
+    LOWER,
+    RAISE,
+    PHI,
+    FN,
+};
+
+#define XXXX KC_NO
+#define ____ KC_TRNS
```

Comment:
If you use the QMK-defined aliases for these keycodes, you can delete these lines.

https://github.com/qmk/qmk_firmware/blob/157319fbd03e6f7a6618111fd03ab3e53317893e/quantum/quantum_keycodes.h#L29-L31

Reply:
I actually really like these shorter ones, especially since most keycodes have less than 7 chars.

Reply:
Ah, usually when we see these, the person who has them doesn't know that the native ones exist. It's fine. I also meant to mark this as a general comment instead of a change request. Oops.